sdcm/sct_events/base.py

```diff
@@ -210,6 +221,103 @@ def __del__(self):
                 print(warning)
 
 
+class InformationalEvent(SctEvent, abstract=True):
+
+    def __init__(self, severity: Severity = Severity.UNKNOWN):
+        super(InformationalEvent, self).__init__(severity=severity)
+        self.time_interval = EventInterval.Informational.value
+
+
+class ContinuousEvent(SctEvent, abstract=True):
+
+    def __init__(self, severity: Severity = Severity.UNKNOWN):
+        super().__init__(severity=severity)
+        self.log_file_name = None
+        self.errors = None
+        self.parent_id = None
+
+    def __enter__(self):
+        event = self.begin_event()
+        event.publish()
+        return event
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        if exc_type is None:
+            event = self.finish_event()
+            event.publish()
+            return
+
+    @property
+    def errors_formatted(self):
+        return "\n".join(self.errors) if self.errors is not None else ""
+
+    @property
+    def msgfmt(self):
+        if self.parent_id is not None:
+            return f"{super().msgfmt} parent_id={self.parent_id}"
+
+        return super().msgfmt
+
+    def begin_event(self,
```

Comment:
add todo to rename after refactor will be done

Reply:
done

Reply:
fixed